src/Image.php

```diff
@@ -409,32 +422,42 @@
  */
 class Image extends ImageAutodoc implements \ArrayAccess
 {
+
     /**
-     * Set true when we are logging actions.
+     * The logger instance.
      *
-     * @internal 
+     * @var LoggerInterface
      */
-    static private $_enable_logging = false;
+    private static $logger;
```

Comment:
I guess the README should note that we expect to pass `phpcs --standard=PSR2 src` or removing the `_` will cause warnings.